README.md

```diff
@@ -11,6 +11,34 @@ Source of truth for CDK code generation.
 - `@aws-cdk/service-spec-types` - loading, saving, querying and diffing a db
 - `@aws-cdk/aws-service-spec` - the actual db file
 
+## Data Sources
+
+The data is read iteratively from various sources. Information from later sources adds on to, or replaces, information
+from older sources. 
+
+* **Properties are added**: new properties are added into existing resources and type definitions. Existing properties (and attributes)
+  will never be removed.
+* **Property type information is added**: when a new type is found, the old type is moved to the `previousTypes` array.
+  However, for backwards compatibility reasons, CDK will currently (only) render the oldest type it can find. Newer types can
+  be also rendered in the future, but are not right now. In a model diff, type history is rendered with a `⇐` between them, in order
+  from new to old. If you see `type1 ⇐ type2 ⇐ type3`, then `type3` is the oldest and is the one that will be used by CDK.
+* **Other property attributes are overwritten**: things like documentation, optionality, etc. are overwritten, so newer specifications
+  fully overwrite the information from older specifications.
+
+Sources are read in this order:
+
+| What | Description | Updates |
+|------|-------------|--------------------
+| Resource Spec | This is the original [CloudFormation Resource Specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html), which is being replaced by the Registry Schema. Imported in order from `us-east-1`, `us-west-2`. | Frozen at version `144.0.0` (Oct 13, 2023). |
```

Comment:
Does this refer to the files in the [cfnspec/spec-source/specification/000_cfn/000_official][1] folder? I can see version 130.1.0 there.

[1]: https://github.com/aws/aws-cdk/tree/main/packages/%40aws-cdk/cfnspec/spec-source/specification/000_cfn/000_official

Reply:
It does not. It's the copy in this repository.

Reply:
The spec in `aws/aws-cdk` is about to go away. Will need a week or two to complete that work.